src/lava/lib/optimization/solvers/generic/processes.py

```diff
@@ -0,0 +1,290 @@
+# Copyright (C) 2021 Intel Corporation

```

Comment:
It seems like this file is mostly code duplication from other files? See some examples below.